test/acceptance/features/steps/steps.py

```diff
@@ -303,27 +304,55 @@ def quarkus_app_is_imported_as_knative_service(context, application_name):
     context.application_type = "knative"
 
 
-# STEP
+def check_result(secret_name):
+    check = secret_name is not None
+    return check
+
+
+def get_sbr_secret_name(context):
+    openshift = Openshift()
+    output = ""
+    if "sbr_name" in context:
+        json_path = "{.status.secret}"
+        output = openshift.get_resource_info_by_jsonpath("servicebindings", context.sbr_name, context.namespace.name, json_path)
+    assert output is not None, "Failed to fetch secret name from ServiceBinding"
+    return output
+
+
+def check_secret_update(context):
+    output = get_sbr_secret_name(context)
+    if context.sb_secret == output:
+        return None
+    context.sb_secret = output
+    return output
+
+
+@step(u'Secret should be updated')
+def then_secret_should_be_updated(context):
+    polling2.poll(lambda: check_secret_update(context), step=100, timeout=1000, ignore_exceptions=(ValueError,), check_success=check_result)
 
 
-@then(u'"{app_name}" deployment must contain SBR name "{sbr_name}"')
+# STEP
+@then(u'"{app_name}" deployment must contain reference to secret existing in service binding "{sbr_name}"')
```

Comment:
`sbr_name` is not used in the function at all, so we can remove it from the step and the function argument list.

Reply:
Done